### DISCUSSION TITLE
Authentifizierung am WebInterface

### DISCUSSION THREAD

**Discussion by Rilnaista** [jomjol/AI-on-the-edge-device#2501] [Q&A]
Hallo, ich hab bereits gesehen, dass es das Feature auf der FeatureRequest List gibt und das für eine "Paid-Version" zurückgehalten wird. Ich sag mal - daran soll es vermutlich nicht scheitern. Was soll denn die Paid Version kosten und wo kann man sie beziehen? Danke für eine Rückmeldung! Viele Grüße Florian

**Comment by caco3** [comment]
Es ist kein Bezahlfeature dafür geplant.

Wir haben als Entwickler einfach entschieden, dass wir keine (trügerische) Sicherheit vorgaukeln wollen und deshalb keine Authentifizierung anbieten wollen.
Eine saubere, sichere Authentifizierung ist auf so einem Gerät von er Leistung her sowieso fast nicht machbar.

**Reply by Rilnaista** [reply]
Es geht ja nicht um trügerische Sicherheit. Es geht darum, dass nicht jeder Gast im eigenen WLAN einfach die Oberfläche öffnen und Einstellungen ändern kann. Z.B. würde es vollkommen reichen, wenn man ein "lock"-File oder ähnliches auf der SD ablegen könnte und damit die Weboberfläche gar nicht erst geladen würde.

Das würde auch keine "Sicherheit" vorgaukeln, das würde funktionieren.

Wäre es mit der Lizenz zulässig den Code selbst derart anzupassen?

**Reply by caco3** [reply]
Ja, du kannst es forken und modifizieren, so wie Du wills, solange Du dann damit nicht Geld verdienst.
